### DISCUSSION TITLE
Open SDG 2.0.0-beta2 ready for testing

### DISCUSSION THREAD

**Discussion by brockfanning** [open-sdg/open-sdg#1690] [Announcements]
Dear members of the Open SDG community,

We have released 2.0.0-beta1 of Open SDG! If you are in a position to test it out, we would greatly appreciate any feedback. To that end, here are links to drafts of the relevant documentation:

* [Updates](https://github.com/brockfanning/open-sdg/blob/2.0.0-release-docs/docs/updates.md)
* [Changelog](https://github.com/brockfanning/open-sdg/blob/2.0.0-release-docs/docs/changelog.md)
* [Upgrade instructions](https://github.com/brockfanning/open-sdg/blob/2.0.0-release-docs/docs/upgrades/upgrading-2-0-0.md)

This is our first major upgrade and we're excited to hear what you think. Thank you!

**Comment by brockfanning** [comment]
Tagging some folks that I believe may be in a position to test: @MoBosse @christoph-LSN @otis-bath

**Comment by christoph-LSN** [comment]
@brockfanning everthing is working fine on my IPhone. Only the legend entries are not shown on the map

https://christoph-lsn.github.io/MT_Site/3-1-2/

![Bildschirmfoto 2022-06-21 um 10 20 33](https://user-images.githubusercontent.com/94443931/174752734-dc2fa4a6-d137-4a10-abb5-e86ead21319a.png)

**Reply by brockfanning** [reply]
@christoph-LSN Sounds good, thank you for testing. The issue with the map legend is to be expected - that legend is hidden on small screens since it takes up so much space.

**Comment by MoBosse** [comment]
@brockfanning Great to hear that 2.0 is ready! I hope to give some feedback next week. 
One thing that is on my mind right now: Is it difficult to add some sort of alert message that pops up when a user accesses the pages with Internet Explorer? I think the pages load with IE but don't display any data because Chartjs 3 is not supported, which would be a bit confusing.

**Reply by brockfanning** [reply]
@MoBosse Thanks very much! No I don't think that would be difficult. I know that targeting IE10/11 is a bit different than IE9 or less, but anyway they can all be done.

**Comment by christoph-LSN** [comment]
@brockfanning I am not sure if this is an 2.0.0-dev related thing:

If I go mouseover a line or barchart values are shown. The thousends delimiter "," is shown... in Germany we have a "." like this 1.000

example indicator: https://christoph-lsn.github.io/MT_Site/1-2-1/

![image](https://user-images.githubusercontent.com/94443931/175039660-2d0d6d98-b2fc-42d4-a41c-723451830342.png)

**Reply by christoph-LSN** [reply]
@brockfanning @MoBosse 

only in this indicator the thousands delimiter is correct https://christoph-lsn.github.io/MT_Site/1-1-1/

![image](https://user-images.githubusercontent.com/94443931/175520048-38c2df61-5cc8-48e1-8ad4-413346672be2.png)

If there are decimal sperated values like in this indicator https://christoph-lsn.github.io/MT_Site/1-1-2/ the tool tip shows three digits after the colon

![image](https://user-images.githubusercontent.com/94443931/175520251-27f9d746-d99b-4e63-8b72-3cdb5c0e5574.png)

**Comment by christoph-LSN** [comment]
@brockfanning @MoBosse 


0,0 values on the map are shown  as "  "
Example indicator: https://christoph-lsn.github.io/MT_Site/1-1-3/ (select "Friesland" in Gebietseinheit)
![image](https://user-images.githubusercontent.com/94443931/175051892-a66ad0d8-15a9-4be4-8a24-0557f16dbfc0.png)

![image](https://user-images.githubusercontent.com/94443931/175052290-6fd6d539-ae47-4feb-9486-d0ef9a93ecbc.png)

**Comment by brockfanning** [comment]
@christoph-LSN I don't have a quick fix for the thousands issue - that seems like something Chart.js might be doing.

But the issue with the zeroes is fixable. Do you have time to test out this pull-request? https://github.com/open-sdg/open-sdg/pull/1705

**Comment by brockfanning** [comment]
Quick update here - we've released a "2.0.0-beta2" tag and updated the docs:

* [Updates](https://github.com/brockfanning/open-sdg/blob/2.0.0-release-docs/docs/updates.md)
* [Changelog](https://github.com/brockfanning/open-sdg/blob/2.0.0-release-docs/docs/changelog.md)
* [Upgrade instructions](https://github.com/brockfanning/open-sdg/blob/2.0.0-release-docs/docs/upgrades/upgrading-2-0-0.md)

**Comment by brockfanning** [comment]
Thanks all for the testing, we've now released 2.0.0!
* [Updates](https://open-sdg.readthedocs.io/en/latest/updates/)
* [Changelog](https://open-sdg.readthedocs.io/en/latest/changelog/)
* [Upgrade instructions](https://open-sdg.readthedocs.io/en/latest/upgrades/upgrading-2-0-0/)
